app/Models/MessageHistory.php

```diff
@@ -35,4 +35,22 @@ public function scopeDataChart($query, $statisticParams, $status)
             ->orderBy('date', 'ASC')
             ->pluck('quantity', 'date');
     }
+
+    public function scopeDataChartByUser($query, $statisticParams, $status, $userID)
+    {
+        $from = date('Y-m-d 00:00:00', strtotime($statisticParams['fromDate']));
+        $to = date('Y-m-d 23:59:59', strtotime($statisticParams['toDate']));
+
+        return $query
+            ->select(DB::raw('DATE_FORMAT(message_histories.created_at, "%d-%m-%Y") as date'),
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket</br>**Severity**: error</br>**File**: app/Models/MessageHistory.php</br>**Line**: 45</br>Opening parenthesis of a multi-line function call must be the last content on the line